examples/todomvc/components/itemview.go

```diff
@@ -52,6 +53,7 @@ func (p *ItemView) onStartEdit(event *vecty.Event) {
 	p.editing = true
 	p.editTitle = p.Item.Title
 	p.ReconcileBody()
+	p.input.Node().Call("focus")
```

Comment:
yeah, because `onStartEdit` is only called by an event handler which is itself registered on double click within the `render` method itself.